Model/CreditCardPaymentMethod.php

```diff
@@ -133,7 +133,12 @@ private function _addThreeDLimits($oPayment, &$oCreditCardConfig)
      */
     private function _convertAmountCurrency($fAmount, $fFromFactor, $oToCurrency)
     {
-        return Registry::getUtils()->fround($fAmount / $fFromFactor * $oToCurrency->rate, $oToCurrency);
+        $fDivisor = $fFromFactor * $oToCurrency->rate;
+        if ($fDivisor === 0.0) {
```

Comment:
👍

Reply:
Actually now I thought that those are floats. Maybe we should prefer a condition like `abs($fDivisor) < $fEpsilon`, where epsilon is a very small number?

Reply:
but the division will work with every float except if it is exactly *0* 🤔